CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 ### Changed
 
 - Upgrade `react-phone-number-input` to v3.1.38
+- Revert change to return document type as 'unknown' in `onComplete` callback payload if Residence Permit selected now that the API supports that document type for document uploads
```

Comment:
I got the wrong idea when I read this. My interpretation was that this change would make document type to return 'unknown' in `onComplete`, when in fact (the code shows) it is the opposite. I would suggest rephrasing it to make it clearer:
```suggestion
- Revert change which returns document type as 'unknown' in `onComplete` callback payload if Residence Permit selected. API now supports that document type for document uploads.
```